tests/test_processing_stack.py

```diff
@@ -165,3 +167,21 @@ def should_successfully_run_dataset_version_creation_process(
                 time.sleep(5)
 
             assert copy_job["Job"]["Status"] == "Complete", copy_job
+
+            with subtests.test(msg="Import Status Endpoint"):
```

Comment:
This test is getting so complicated that we should probably split it into three:

- Should finish the state machine successfully
- Should complete S3 batch copy
- Should report import status after success